spirv_glsl.hpp

```diff
@@ -51,6 +51,13 @@ struct PlsRemap
 	PlsFormat format;
 };
 
+struct InterfaceVariableRename
+{
+	std::string in_or_out;

```

Comment:
Can probably be replaced with spv::StorageClass, and in or out can be validated earlier.